src/main/scala/PrivateHome/Devices/MHz/mhzSwitch.scala

```diff
@@ -41,23 +44,32 @@ case class mhzSwitch(setupID: String, _keepStatus: Boolean, private var _systemC
   /**
    * Turns the Switch on
    */
-  override def on(): Unit = queue.queue.enqueue(mhzCommand(_systemCode, _unitCode, command = true))
+  override def on(): Unit = sendMhz(mhzCommand(_systemCode, _unitCode, command = true))
 
   /**
    * Turns the Switch off
    */
-  override def off(): Unit = queue.queue.enqueue(mhzCommand(_systemCode, _unitCode, command = false))
+  override def off(): Unit = sendMhz(mhzCommand(_systemCode, _unitCode, command = false))
 
   /**
    * Generates a XML Node containing all Information to Regenerate the object
+   *
    * @return The Node Object
    */
   override def toXml: Node = <switch>
```

Comment:
Please use <switch id ="{id}">. Otherwise editXML.scala wouldn't work correctly.

Reply:
What did you mean with .?

Reply:
@maximiliani

Reply:
@maximiliani

Reply:
@maximiliani

Reply:
In toXml you used <switch>! This causes problems in the editXML.scala! Please use <switch id="{id}">! Thanks

Reply:
In toXml you used //<switch//>! This causes problems in the editXML.scala! Please use //<switch id //= //"id//"//> ! Thanks